pom.xml

```diff
@@ -39,7 +39,7 @@
 
 		<!-- runtime dependencies -->
 		<api.version>0.1.3</api.version>
-		<secret-service.version>1.2.1</secret-service.version>
+		<secret-service.version>1.3.0</secret-service.version>
```

Comment:
1.3.0 is still buggy (as is 1.4.0). This needs to be changed once a new release is out (1.4.x)

Reply:
There is `1.5.0`